pkg/reconciler/revision/resources/deploy.go

```diff
@@ -50,6 +50,25 @@ var (
 		SubPathExpr: "$(K_INTERNAL_POD_NAMESPACE)_$(K_INTERNAL_POD_NAME)_",
 	}
 
+	varProjectedVolume = corev1.Volume{
+		Name: "knative-proj-volume",
+		VolumeSource: corev1.VolumeSource{
+			Projected: &corev1.ProjectedVolumeSource{
+				Sources: []corev1.VolumeProjection{{
+					ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+						ExpirationSeconds: ptr.Int64(600),
+						Path:              "token",
+						Audience:          "freezer"},
+				}},
+			},
+		},
+	}
+
+	varProjectedVolumeMount = corev1.VolumeMount{
+		Name:      varProjectedVolume.Name,
```

Comment:
being very pedantic, but Im wondering a bit about the name of the volume being `knative-proj-volume` (which implies it's a generic projected volume), but then it gets mounted to `/var/run/secrets/tokens` (which suggests it's just for tokens). Maybe the volume should be named `knative-token-volume` actually?

Reply:
Sounds good. I'll also update the variable names to `varTokenVolume` and `varTokenVolumeMount`